bin/worker

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env node
```

Comment:
How are the workers started now? In the `micros` branch, I have added some code to `run.js` to optionally start workers from the same process that starts web workers.

Reply:
Ah, found the answer in the `package.json`: `start:worker`